### ISSUE TITLE
Field import/export css and js 404 not found

### ISSUE THREAD

**Issue by paulthecoder** [reduxframework/redux-framework#2214]
Redux 3.5.1
WordPress 4.1.1
Dev mode: off

```
http://localhost:8888/wp-content/plugins/redux-framework/ReduxCore/inc/extensions/import_export/import_export/import_export/field_import_export.min.js?ver=4.0

http://localhost:8888/new_woo/wp-content/plugins/redux-framework/ReduxCore/inc/extensions/import_export/import_export/import_export/field_import_export.css?ver=1

Failed to load resource: the server responded with a status of 404 (Not Found)
```
This is whats showing up in my google chrome dev tools. Just updated to redux 3.5.1 and this happened. I'm looking for any causes on my end, will update if I find anything.

**Comment by kprovance** [comment]
Well, I did fix it.  Looks like someone else broke it.  ::sigh::

**Comment by kprovance** [comment]
Fixed in v3.5.2

**Comment by paulthecoder** [comment]
Awesome, thanks!
